core/vibes/soul/sections/not-found/index.tsx

```diff
@@ -8,7 +8,7 @@ export function NotFound({
   subtitle = "Take a look around if you're lost.",
 }: Props) {
   return (
-    <section className="@container">
+    <section className="@container pb-20">
```

Comment:
This change has a high risk of being blasted a way by pulling in updates from VIBES. We might want to handle it elsewhere unless you think that we should make this change upstream as well @bookernath?